rel-eng/tito.props

```diff
@@ -4,7 +4,7 @@
 #
 
 [buildconfig]
-builder = tito.builder.GitAnnexBuilder
+builder = obal.tito_extensions.git_annex_spec_builder.GitAnnexSpecBuilder
```

Comment:
This might be me nitpicking, but I'd keep this in a separate commit since it's unrelated to the actual bumping of the celery stack.

Reply:
Might be nitpicky, but definitely fair.  I intended to to such, and it got swept up in the git add . && git commit --amend

Reply:
Updated